src/panels/lovelace/cards/hui-map-card.ts

```diff
@@ -308,47 +321,47 @@ class HuiMapCard extends LitElement implements LovelaceCard {
     return color;
   }
 
-  private _getEntities = memoizeOne(
-    (
-      states: HassEntities,
-      config: MapCardConfig,
-      configEntities?: MapEntityConfig[]
-    ) => {
-      if (!states || !config) {
-        return undefined;
+  private _getSourceEntities = memoizeOne(
```

Comment:
This should not be memoized, as you are already storing it on the class

Reply:
Not storing it on the class anymore (removed the class member), so keeping it memoized.